src/main/java/seedu/contax/ui/TextStyleHelper.java

```diff
@@ -0,0 +1,106 @@
+package seedu.contax.ui;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontPosture;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.Text;
+
+public class TextStyleHelper {
+
+    /**
+     * Parser to identify bold/italic components within a string
```

Comment:
I think this is no longer accurate. Also, the javadoc should terminate with a full stop.

Reply:
thanks for the catch, resolved in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/207/commits/5712c55b1563e8adfdcf92abef182a1197e32964